test/e2e/volumes.go

```diff
@@ -285,6 +304,11 @@ func injectHtml(client *client.Client, config VolumeTestConfig, volume api.Volum
 					},
 				},
 			},
+			SecurityContext: &api.PodSecurityContext{
```

Comment:
I'm wondering if this test should be tagged with SELinux.  It's debatable; I don't think I would normally want to run this with 'stuff that exercises core SELinux functionality in kube', but I definitely want to test it when I run absolutely everything that touches SELinux.